q2_types/sample_data/tests/test_transformer.py

```diff
@@ -26,6 +26,10 @@ def test_pd_series_to_alpha_diversity_format(self):
                         name='shannon', index=exp_index)
 
         obs = transformer(exp)
+        '''
```

Comment:
This is a [docstring](https://www.python.org/dev/peps/pep-0008/#documentation-strings), which is a certain type of [comment](https://www.python.org/dev/peps/pep-0008/#comments). Typically a docstring is "[a string literal that occurs as the first statement in a module, function, class, or method definition](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)". Can you please turn this comment into either a block comment, or an inline comment?